docs/device-management/5-ota/README.md

```diff
@@ -4,33 +4,75 @@ title: Over-the-Air (OTA) Updates Overview
+binary bundles, enabling updates for devices as diverse as smart speakers,
+digital signage, machine learning enabled sensor systems, multiple processor
+embedded devices, and more.
 
-Our DFU feature is highly flexible, allowing to not just update your main firmware, but sub components like the Cellular Modem firmware, update a on device Machine Learning model or even sending custom binaries required for your application as part of a release.
+Our OTA feature is highly flexible, allowing to not just update your main
+firmware, but sub components like the Cellular Modem firmware, update a on
+device Machine Learning model or even sending custom binaries required for your
+application as part of a release.
 
-Artifact and Releases can be accessed through either the [`goliothctl dfu`](/reference/command-line-tools/goliothctl/goliothctl_dfu) subcommand, the [DFU REST API](/reference/rest-api/openapi), or from Golioth Web Console.
+Artifact and Releases can be accessed from Golioth Web Console, the [Golioth REST
+API](/reference/rest-api/openapi), or by using the [`goliothctl
+dfu`](/reference/command-line-tools/goliothctl/goliothctl_dfu) subcommand.
 
-> See [CoAP Gateway Interface](/reference/protocols/coap/ota)
+:::info Details for OTA manifest, path, and status
 
-### Concepts
+For more information on how the Golioth server and IoT devices communicate with
+each other about Over-the-Air updates, see the OTA section of our CoAP
+reference:
 
-- Artifacts
-  - An artifact is a binary image that can be pushed to your embedded device. This may take the form of the base level firmware (to be used with a bootloader), a binary blob for an external processor, a firmware image for the modem onboard (such as the nRF91), or a required content element for the running application, such as an image or sound file. Individual binaries (artifacts) can be versioned and tracked before being added to a release.
-- Release
-  - A release is a collection of artifacts to be delivered to the target embedded device. In the simplest example, a release will only contain a single artifact, the firmware image to reprogram the main processor on board. Once you mark a release as active, any eligible embedded devices will attempt to download and install that firmware update. Release rollout and rollback can be controlled from the [Golioth Console](https://console.golioth.io)
-- Blueprint
-  - Blueprint is a mapping of the elements on board your system. This includes the memory mapping of flash elements onboard. Blueprints help to segment which devices in your fleet are elible to download a particular release. For instance, the release may have a different firmware memory offset for one blueprint vs another, due to differences in the flash memory on different versions of a product.
+- [**CoAP Gateway Interface**](/reference/protocols/coap/ota)
 
-### Use Cases
+:::
 
-Here are some ideas on what can be created using our DFU Service.
+## Concepts
```

Comment:
I know these concepts are not new to this PR, but it might be worth adding tags as a concept here, as they are, similarly to blueprints, used to filter the devices which will receive a given release.